modules/zivid/handeye.py

```diff
@@ -0,0 +1,125 @@
+"""Contains hand-eye calibration functions and classes."""
+import _zivid
+
+
+def detect_feature_points(point_cloud):
+    """Detect feature points from a calibration object in a point cloud.
+
+    The functionality is to be exclusively used in combination with Zivid verified checkerboards.
+    For further information please visit Zivid help page.
+
+    Args:
+        point_cloud: cloud from a frame containing an image of a calibration object
+
+    Returns:
+        Instance of DetectionResult
+
+    """
+    return DetectionResult(
+        _zivid.handeye.detect_feature_points(
+            point_cloud._PointCloud__impl  # pylint: disable=protected-access
+        )
+    )
+
+
+def calibrate_eye_in_hand(calibration_inputs):
+    """Perform eye-in-hand calibration.
+
+    The procedure requires feature point sets acquired at the minimum from two poses.
+    All the input poses have to be different. The feature point sets cannot be empty.
+    All the feature point sets have to have same number of feature points.
+    An exception will be thrown if the above requirements are not fulfilled.
+
+    Args:
+        calibration_inputs: a Sequence of CalibrationInput instances
+
+    Returns:
+        Instance of CalibrationOutput
+
+    """
+    return _zivid.handeye.calibrate_eye_in_hand(
+        [
+            calib._CalibrationInput__impl  # pylint: disable=protected-access
+            for calib in calibration_inputs
+        ]
+    )
+
+
+def calibrate_eye_to_hand(calibration_inputs):
```

Comment:
I've locally recreated the corresponding C++ CLI-tool for this one (the one that takes real poses and pictures from files), and confirmed that the resulting transform is the same given the same input-data.

Reply:
Can we expect to have this CLI to ship it to the Customer?

Reply:
We should definitely consider it.